mlflow/utils/search_utils.py

```diff
@@ -869,11 +869,11 @@ class SearchExperimentsUtils(SearchUtils):
 
     @classmethod
     def _invalid_statement_token_search_experiments(cls, token):
-        if isinstance(token, Comparison):
-            return False
-        elif token.is_whitespace:
-            return False
-        elif token.match(ttype=TokenType.Keyword, values=["AND"]):
+        if (
```

Comment:
comment: [this comment](https://github.com/mlflow/mlflow/pull/9919#discussion_r1356516598) is actually incorporated in this PR